src/main/java/org/springframework/data/elasticsearch/core/ReactiveIndexOperations.java

```diff
@@ -284,5 +286,6 @@
 	 */
 	IndexCoordinates getIndexCoordinates();
 
+	Mono<List<IndexInformation>> getInformation();
```

Comment:
we should rather return a `Flux<IndexInformation>` than a Mono of a list here.
Please add Javadoc and `@since 4.2`